cli/version/version.go

```diff
@@ -54,14 +54,21 @@ func BuildCliVersionString() string {
 }
 
 func PrintVersionString(projectPath string, projectPathIsSet bool, showRocksVersions bool) error {
+	var rocksVersions common.RocksVersions
+	var err error
+
 	fmt.Println(BuildCliVersionString())
 
-	if err := printCartridgeVersion(projectPath); err != nil {
-		currentErrorString := cartridgeVersionGetError
-		if showRocksVersions {
-			currentErrorString = rocksVersionsGetError
-		}
+	currentErrorString := cartridgeVersionGetError
+	if showRocksVersions {
+		currentErrorString = rocksVersionsGetError
+	}
 
+	if rocksVersions, err = LuaGetRocksVersions(projectPath); err != nil {
+		return fmt.Errorf("%s: %s", currentErrorString, err)
+	}
+
+	if err := printCartridgeVersion(projectPath, rocksVersions, showRocksVersions); err != nil {
 		if projectPathIsSet {
 			return fmt.Errorf("%s: %s", currentErrorString, err)
```

Comment:
Seems that for `printCartridgeVersion` this is always `cartridgeVersionGetError`.